emsdk.py

```diff
@@ -2026,14 +2024,12 @@ def install_tool(self):
       success = git_clone_checkout_and_pull(url, self.installation_path(), self.git_branch)
     elif url.endswith(ARCHIVE_SUFFIXES):
       # TODO: explain the vs-tool special-casing
-      download_even_if_exists = (self.id == 'vs-tool')
       # The 'releases' sdk is doesn't include a verion number in the directory
       # name and instead only one version can be install at the time and each
       # one will clobber the other.  This means we always need to extract this
       # archive even when the target directory exists.
-      download_even_if_exists = (self.id == 'releases')
       filename_prefix = getattr(self, 'zipfile_prefix', '')
-      success = download_and_unzip(url, self.installation_path(), download_even_if_exists=download_even_if_exists, filename_prefix=filename_prefix)
+      success = download_and_unzip(url, self.installation_path(), (self.id == 'vs-tool') or (self.id == 'releases'), filename_prefix=filename_prefix)
```

Comment:
Why no keep the sperate variable?  I think its make the code a little clearer, and to explains the `TODO: explain the vs-tool special-casing` above which.  Why do you special vs-tool and releases? 

Maybe just change the `=` to and `|=` in  `download_even_if_exists = (self.id == 'releases')` above?

Reply:
I guess this is one on the bugfixes that can be landed separately?  I also wonder if we should keep ths vs-tool special casing around given that is seems to have been inactive for a while now. 

In any case you split this change out into a separate PR?